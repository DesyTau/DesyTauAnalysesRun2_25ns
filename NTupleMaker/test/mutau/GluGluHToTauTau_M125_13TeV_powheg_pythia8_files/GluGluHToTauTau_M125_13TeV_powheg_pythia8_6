/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_6.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_7.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_78.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_79.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_80.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_81.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_82.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_83.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_86.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_87.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_88.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_89.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_90.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_91.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_92.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_93.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_94.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_95.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_96.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_97.root
