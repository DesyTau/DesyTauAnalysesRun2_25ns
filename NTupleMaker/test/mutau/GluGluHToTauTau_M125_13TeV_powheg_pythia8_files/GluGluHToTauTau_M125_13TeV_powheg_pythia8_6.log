000 (2509991.000.000) 01/17 11:47:05 Job submitted from host: <131.169.223.40:9618?addrs=131.169.223.40-9618+[2001-638-700-10df--1-28]-9618&noUDP&sock=7812_f5f0_32>
...
022 (2509991.000.000) 01/17 11:48:54 Job disconnected, attempting to reconnect
    Socket between submit and execute hosts closed unexpectedly
    Trying to reconnect to slot1@bird144.desy.de <131.169.84.105:35168?addrs=131.169.84.105-35168>
...
024 (2509991.000.000) 01/17 11:48:54 Job reconnection failed
    Job not found at execution machine
    Can not reconnect to slot1@bird144.desy.de, rescheduling job
...
001 (2509991.000.000) 01/17 11:49:36 Job executing on host: <131.169.163.107:46494?addrs=131.169.163.107-46494>
...
006 (2509991.000.000) 01/17 11:49:45 Image size of job updated: 156064
	98  -  MemoryUsage of job (MB)
	99936  -  ResidentSetSize of job (KB)
...
006 (2509991.000.000) 01/17 11:49:56 Image size of job updated: 206712
	98  -  MemoryUsage of job (MB)
	99968  -  ResidentSetSize of job (KB)
...
005 (2509991.000.000) 01/17 11:49:56 Job terminated.
	(1) Normal termination (return value 255)
		Usr 0 00:00:14, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:14, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	0  -  Run Bytes Sent By Job
	0  -  Run Bytes Received By Job
	0  -  Total Bytes Sent By Job
	0  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :     1.00        1         1 
	   Disk (KB)            :     1     3072000   3919526 
	   Memory (MB)          :    98        2048      2048 
...
