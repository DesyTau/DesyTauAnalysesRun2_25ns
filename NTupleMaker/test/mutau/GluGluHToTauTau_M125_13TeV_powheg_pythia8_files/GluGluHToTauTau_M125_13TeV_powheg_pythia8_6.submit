+RequestRuntime=10000

RequestMemory = 2000

executable = GluGluHToTauTau_M125_13TeV_powheg_pythia8_6.sh

transfer_executable = True
universe            = vanilla
getenv              = True
Requirements        = OpSysAndVer == "SL6"

output              = GluGluHToTauTau_M125_13TeV_powheg_pythia8_6.out
error               = GluGluHToTauTau_M125_13TeV_powheg_pythia8_6.error
log                 = GluGluHToTauTau_M125_13TeV_powheg_pythia8_6.log

queue

