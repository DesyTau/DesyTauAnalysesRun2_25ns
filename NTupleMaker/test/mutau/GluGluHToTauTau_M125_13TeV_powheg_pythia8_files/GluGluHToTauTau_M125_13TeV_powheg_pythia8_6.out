/nfs/dust/cms/user/cardinia/HtoTauTau/HiggsCP/CMSSW_9_4_9/src/DesyTauAnalyses/NTupleMaker/test/mutau/GluGluHToTauTau_M125_13TeV_powheg_pythia8_files
Number of single lepton trigger legs = 2
Number of X trigger legs (lep leg)   = 2
Number of X trigger legs (tau leg)   = 2
Number of ditau trigger legs         = 0
dxyLeptonCut 0.045
dzLeptonCut 0.2
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_6.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_7.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_78.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_79.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_80.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_81.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_82.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_83.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_86.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_87.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_88.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_89.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_90.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_91.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_92.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_93.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_94.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_95.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_96.root
/nfs/dust/cms/user/tlenz/13TeV/2017/NTuples/MC/htt_signal_htxs_2017/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1/GluGluHToTauTau_M125_13TeV_powheg_pythia8_ext1_97.root
Histogram GluGluHToTauTau_M125_13TeV_powheg_pythia8_pileup is not present in pileup file
